airflow/jobs/scheduler_job_runner.py

```diff
@@ -213,6 +214,20 @@ def _exit_gracefully(self, signum: int, frame: FrameType | None) -> None:
             self.processor_agent.end()
         sys.exit(os.EX_OK)
 
+    def _log_memoery_usage(self, signum: int, frame: FrameType | None) -> None:
```

Comment:
Small typo here, otherwise looking good
```suggestion
    def _log_memory_usage(self, signum: int, frame: FrameType | None) -> None:
```